helmfile.d/stacks/tekton.yaml

```
@@ -1,5 +1,23 @@
 ---
 templates:
+  tekton:
+    namespace: tekton-pipelines
+    labels:
+      app: tekton-pipelines
```

```suggestion
      app: tekton
```

And make it so that psp and tekton itself inherits it as well.